### ISSUE TITLE
Add Creation and Closing Date to Standard-Format.

### ISSUE THREAD

**Issue by gerlowskija** [gerlowskija/bugging-out#22]
Currently, the supported standard-format is extremely limited.

I'd like to expand it by adding the creation-date, as well as the date-resolved.  These are pretty high-value pieces of information to add, as it is the basis for any sort of question of the form "How did community behavior X change over time?"

As acceptance criteria for this issue, exported data should look like:

    <issue number="DATA">
        <creation creation-date="DATA"/>
        <resolution resolution-date="DATA"/>
    </issue>

**Comment by gerlowskija** [comment]
The <creation> tag is there.  Now to add the "resolution" tag.

Resolution might be a bit trickier because not all issues have been resolved.  But it shouldn't be too difficult.

**Comment by gerlowskija** [comment]
This story is now closed.  However, the standard format has been modified slightly.

The new format looks like:

    <issue number="DATA">
        <creation creation-date="DATA"/>
        <resolution resolution-date="DATA" resolved="closed|open"/>
    </issue>

When resolved="open", then there is no resolution-date attribute on the resolution tag.

**Comment by gerlowskija** [comment]
The test suite for bugging-out uses data exported from various Github issues for making sure the right data is exported, as starting data for conversion testing, etc.

One of the issues relied upon in the test data is this one.  Since I recently changed the default export format for Github I recently  had to open this issue so it could be re-exported in an 'open' state.

Now that the test data has been updated, I will re-close this issue.
